src/validation/loginValidation.js

```diff
@@ -0,0 +1,8 @@
+import Joi from '@hapi/joi';
+
+const loginSchema = Joi.object({
+  email: Joi.string().email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } }),
+  password: Joi.string().pattern(new RegExp('^[a-zA-Z0-9@]{4,16}$')),
```

Comment:
if think it is not necessary to validate the password

Reply:
Its not necessary actually